lib/views/widgets/navigation/floating_action_button.dart

```diff
@@ -1,16 +1,27 @@
 import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
 import 'package:sun_flutter_capstone/consts/global_style.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:sun_flutter_capstone/utils/routes/router.gr.dart';
 
-class CenterActionButton extends StatelessWidget {
+class CenterActionButton extends HookConsumerWidget {
```

Comment:
@louisepujante Asking so I can also learn 😄 
What is this change for?

Reply:
This was added by Ervin, I think it's to trigger the routing.. same with the bottom nav bar buttons which also uses this widget